pages/apis/webhooks/job_events.md

```diff
@@ -55,3 +56,6 @@ Example request body:
   }
 }
 ```
+
+>📘
+> The job states of webhook events do not return <code>finished</code>, but returns <code>passed</code> or <code>failed</code> based on the  <a href="/docs/apis/rest-api/builds">REST API</a>  for legacy compatibility.  
```

Comment:
I found this a bit confusing. Perhaps it should be in the "Request body data" section? But that section already links you to the REST API objects. I wonder if this could be left out. It doesn't look like any of the customer problems have arisen from confusion from this side of things. What do you think?

```suggestion
```

Reply:
Hi @mbelton-buildkite , yes this is a bit confusing. I am also not quite convinced this is the right place to add this note. But then, we really have no place to describe that a job state has different values for webhook notifications and EventBridge notifications. However, we do mention that they are different for REST API [here](https://buildkite.com/docs/pipelines/defining-steps#job-states).  I am fine not to have this PR merged. But maybe we can have better docs for our notification services integration payloads.

Reply:
I think it would be good to merge the change on the Eventbridge page. From my understanding, that directly addresses the problem the customers ran into. I think it would be best to remove the change to this file for now. What do you think?

Reply:
Thanks for the inputs @mbelton-buildkite . We can close this PR and I'll raise a new one with a note at the Eventbridge notification page.